### PR TITLE
Add Wordpress Fargate RDS example

### PR DESCRIPTION
**Pull request by polkx**
Example based on:
https://github.com/pulumi/examples/tree/master/aws-ts-wordpress-fargate-rds

### CODE REVIEW

File: examples/aws-wordpress-fargate-rds/Frontent.scala [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+              `type` = "forward",
+              targetGroupArn = atg.arn
+            )
+          )
+        )
+      )
+      val role = aws.iam.Role(
+        name = s"$name-task-role",
+        aws.iam.RoleArgs(
+          assumeRolePolicy = p"""{
```
[the remark this answers is not in the document]
Comment (polkx) [reply examples/aws-wordpress-fargate-rds/Frontent.scala]: done

**Review by pawelprazak — APPROVED** [review]

File: examples/aws-wordpress-fargate-rds/Frontent.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (pawelprazak) [reply examples/aws-wordpress-fargate-rds/Frontent.scala]: I'm sorry, I might have not been clear enough in my comment, the changes done use JSON AST,
the ask was to use the JSON interpolator: https://virtuslab.github.io/besom/docs/changelog/#030-16-04-2024

`json"""...""".map(_.prettyPrint)`

**Review by pawelprazak — CHANGES_REQUESTED** [review]
please use `json"""...""".map(_.prettyPrint)`

**Review by lbialy — CHANGES_REQUESTED** [review]
this is JSON AST, json interpolator is `json"{}"`

**Review by lbialy — APPROVED** [review]

**Pull request merged**